data_browser/orm_admin.py

```diff
@@ -44,7 +44,7 @@
     ArrayField = None.__class__
 
 try:
-    from django.fields import JSONField
```

Comment:
this is a fascinating failure on my part that raises all sorts of questions

Reply:
oh this works for me on 3.1 because I have the postgres library installed
but why does it pass on CI

Reply:
I figured it out because I have no postgresql installed and almost all tests failed.

Reply:
Thanks for that, I would not have noticed until someone bug reported it.